lib/src/Uncertainty/StatTests/LinearModelTest.cxx

```diff
@@ -76,12 +78,12 @@ TestResult LinearModelTest::LinearModelFisher(const Sample & firstSample,
   // H0 : Beta_i = 0
   // H1 : Beta_i < 0 or Beta_i > 0
   // The statistics follows a Fisher distribution
-  const Scalar numerator = sumSquaredExplained / (size - df - 1);
-  const Scalar denomerator = sumSquaredResiduals / df;
+  const Scalar numerator = sumSquaredExplained / (size - dof - 1);
```

Comment:
```suggestion
  const Scalar numerator = sumSquaredExplained / (p - 1);
```

Reply:
It is difficult to read. Usually we have `SSM/(p-1)` in literature

Reply:
here you want me to rewrite code I didnt touch, that's for another PR

Reply:
ok I will do the job